content/docs/08-migration-guides/30-migration-guide-5-0.mdx

```diff
@@ -0,0 +1,2355 @@
+---
+title: Migrate AI SDK 4.0 to 5.0
+description: Learn how to upgrade AI SDK 4.0 to 5.0.
+---
+
+// NOTE: change to 4.3?
+// NOTE: need to review titles for clarity and accuracy
+
+# Migrate AI SDK 4.0 to 5.0
+
+<Note>
+  AI SDK 5.0 is a major release that introduces significant improvements in type
+  safety, consistency, and developer experience. This guide will help you
+  migrate from AI SDK 4.0 to 5.0.
+</Note>
+
```

Comment:
note that you may want to wait until GA for production proejects, this is for early adopters who want to helkp uis